arch/Kconfig

```diff
@@ -196,6 +197,25 @@ config SRAM_BASE_ADDRESS
 	  /chosen/zephyr,sram in devicetree. The user should generally avoid
 	  changing it via menuconfig or in configuration files.
 
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_DRAM := zephyr,dram
+
+config DRAM_SIZE
```

Comment:
I don't think you need a Kconfig option since this is being installed in devicetree from the beginning. The legacy SRAM options predate devicetree in Zephyr and are largely around for backwards compat.